### ISSUE TITLE
Synchronisation klappt nicht

### ISSUE THREAD

**Issue by paulamms** [b-lack/qgis-fim-plugin#88]
Es gibt einige Punkte mit dem WF 6, FIM Abgeschlossen, aber ohne eingetragene Daten, sowohl beim Los 2024_01 von EFCA als auch beim Los 2024_07 von Böhlick. Es werden also keine Daten an die DB übergeben, obwohl die Punkte im FIM abgeschlossen sind und auch Daten enthalten. 

Ich habe noch nicht rausfinden können woran es liegt. Ich habe Böhlick gebeten seine QGIS Version upzudaten. EFCA müsste aber auf dem neusten Stand sein. 

Ich kann meine eigenen Punkte (Los 2024_02) aus irgendeinem Grund nicht ins FIM laden, deshalb fällt mir das testen gerade schwer.

**Comment by paulamms** [comment]
Ich kriege das Phänomen mit der nicht funktionieren Synchronisation leider nicht reproduziert. Es scheint aber weiterhin ein Problem zu sein, auch mit QGis update. Nicht begehbare Punkte werden spannender Weise aber regulär synchronisiert.

**Comment by b-lack** [comment]
Wichtig zu wissen wäre, ob sich der Workflow in der Datenbank ändert.

**Comment by paulamms** [comment]
Ja, wie geschrieben, der WF steht in der DB auf 6 (=Abgeschlossen).

**Comment by b-lack** [comment]
Dann könntest du mir doch eigentlich 2 IDs geben, damit man es ausprobieren kann?

**Comment by paulamms** [comment]
Ja stimmt, muss dann nur die Daten wieder raus löschen. Ich schreibe die zwei zu und setzte den Workflow für diese wieder auf 3.

**Comment by paulamms** [comment]
Ich habe dir 522, 534 und 543 zugeschrieben

**Comment by paulamms** [comment]
Nachdem die Werte bei Hase/Biber/Maus von NULL auf FALSE gesetzt wurden klappt bei mir für die Testpunkte die Synchro wieder. Eventuell war das schon alles. 
Ich nehme dir die drei Punkte wieder weg und schließe das Ticket.

**Comment by paulamms** [comment]
Leider werden immernoch Punkte synchronisiert, die leer sind. Mit der Synchro werden auch die von mir vor zwei Tagen auf "FALSE" gesetzten Boolean Parameter (Maus, Hase, Biber) wieder auf "NULL" gesetzt.

**Comment by paulamms** [comment]
Das Problem scheint erst seit September aufzutreten, sowohl bei den Losen von EFCA als auf bei Böhlick

**Comment by paulamms** [comment]
Wir haben einen Formatierungsfehler in den geojson Dateien festgestellt, der den import behindert hat. Mit der neuen Version FIM .30 können diese auch gelesen werden. 
Ich konnte dadurch jetzt die Punkte von "stst" in die DB laden. 

Der eigentliche Synchrofehler muss aber anders passieren, ich habe mit "stst" gesprochen und sie hat die import Funktion nie genutzt.

**Comment by b-lack** [comment]
Tritt das Problem immernoch bei folgenden Punkten 522, 534 und 543 auf? Wird das json,dass du mir geschickt hast nicht korrekt synchronisiert?

Wenn doch, schick mit doch bitte die ID von betroffenen Punkten, besser entsprechende JSON-Dateien.

**Comment by paulamms** [comment]
Es betrifft insgesam ca. 190 Punkte, die sind in der g_los Tabelle anhand Losnr '2024_07',WF 6, Fim Status TRUE, aber leeren Datenfeldern zu erkennen.  

Es kommen täglich ein paar neue hinzu, ich weiß also vorher nicht welche Punkte es betrifft sondern erst, wenn sie schon synchronisiert sind. 

Ich teile hier mal den Ordner mit Exportdateien der ATs, darin befinden sich dann wahrscheinlich auch die leeren Punkte. 
https://faircloud.eu/nextcloud/index.php/s/kdzppzApBGWsmSQ

**Comment by b-lack** [comment]
kannst du mir bitte 1 ID geben bei der das Problem aufgetreten ist. Hilf mir, euch zu helfen.

Ich muss das irgend wie reproduzieren können.

**Comment by paulamms** [comment]
achso, na klar, diese hier zum Beispiel: 

id_g_los = 965, 966, 967  

Ich hab leider keine jsons dazu, die kriege ich wahrscheinlich erst heute nachmittag. Ich weiß aber noch nicht genau für welche Punkte.

**Comment by paulamms** [comment]
In dem oben verlinkten Ordner hat jetzt auch Monique Müller ihre Punkte abgelegt, die sind alle leer synchronisiert worden. Sie hat die Punkte erst nach dem hochladen exportiert.

Alle "hochgeladenen" Punkte weisen im JSON die gleiche Formatierung auf, wie wir sie neulich schonmal entdeckt haben. Überall sind backslashs '\'

**Comment by b-lack** [comment]
Das Plugin wurde (teilweise) nicht als ZIP sondern direkt vom GitHub repo eingebunden. Es handelt sich dabei um einen Arbeitsstand NACH der Version, die in QGIS angezeigt wird. Damit kann die installierte Version nicht mehr nachvollzogen werden, der Fehler kann folglich nicht reproduziert werden.

"Echte" Versionen sind nur hier zu finden, diese können auch nachträglich zum beheben von Fehlern genutzt werden:
Test: https://github.com/b-lack/qgis-fim-plugin/releases
Getestet: https://plugins.qgis.org/plugins/fim/


Also bei jeder Rückmeldung zusätzlich fragen, WIE das plugin installiert wurde.

Um das zukünftig zu unterbinden, kann ich das installieren von github unterbinden!! Wenn die Tests abgeschlossen sind?!
